Readability.js

```diff
@@ -1678,15 +1678,18 @@ Readability.prototype = {
    *
    * @return boolean Whether or not we suspect parse() will suceeed at returning an article object.
    */
-  isProbablyReaderable: function() {
-    var nodes = this._doc.getElementsByTagName("p");
-    if (nodes.length < 5) {
-      return false;
-    }
-
-    var possibleParagraphs = 0;
+  isProbablyReaderable: function(helperIsVisible) {
+    var nodes;
+    if (typeof this._doc.querySelectorAll != "undefined")
+      nodes = this._doc.querySelectorAll("p, pre");
+    else
+      nodes = this._concatNodeLists(this._doc.getElementsByTagName("p"), this._doc.getElementsByTagName("pre"));
+
+    var score = 0;
     for (var i = 0; i < nodes.length; i++) {
```

Comment:
Actually, as we return as soon as we've found something, `_someNode` would be a much more appropriate (and fast) candidate.